requirements.pip

```diff
@@ -0,0 +1,14 @@
+argh==0.26.2
+colorama==0.3.7
+coverage==4.2
+docopt==0.6.2
+pathtools==0.1.2
+pluggy==0.3.1
+py==1.4.31
+pytest==2.9.2
+pytest-cov==2.3.1
+pytest-watch==4.1.0
+PyYAML==3.11
+tox==2.3.1
+virtualenv==15.0.3
+watchdog==0.8.3
```

Comment:
I'd be tempted to allow the setup.py testing extra take care of this.  Remember, you don't actually require any of these dependencies to run your code, only to test it.  So you could skip having a requirements.pip file at all.